server/test/services/export/PdfExporterTest.java

```diff
@@ -32,7 +32,7 @@ public void validatePDFExport() throws IOException, DocumentException {
     String applicantName = "name-unavailable";
     String applicantNameWithApplicationId =
         String.format("%s (%d)", applicantName, applicationOne.id);
-    PdfExporter.InMemoryPdf result = exporter.export(applicationOne);
+    PdfExporter.InMemoryPdf result = exporter.export(applicationOne, false);
```

Comment:
Can you set all the `true` and `false` values in these two test files to look like how you have them on the AdminApplicationController.

```suggestion
    PdfExporter.InMemoryPdf result = exporter.export(applicationOne,  /* showEligibilityText= */ false);
```

Reply:
Sure!